app/views/compute_resources/form/_ovirt.html.erb

```diff
@@ -1,6 +1,7 @@
 <%= text_f f, :url, :size => "col-md-8", :help_block => _("e.g. https://ovirt.example.com/api"), :help_inline => documentation_button('5.2.7oVirt/RHEVNotes') %>
 <%= text_f f, :user, :help_block => _("e.g. admin@internal") %>
 <%= password_f f, :password, :unset => unset_password? %>
+<%= checkbox_f f, :filtered_api, :help_inline => _("If you don't have admin permissions you can use slower filtered API."), :label => _('User level API') %>
```

Comment:
> I would go with whatever terminology is used in the oVirt documentation. 

Out of curiosity, can you send a link to the doc that describes this
term? Because to me, it looks like there is no oVirt documentation at
all. ;-)

-- 
Later,
 Lukas #lzap Zapletal